README.md

```diff
@@ -1,14 +1,16 @@
 # Ledger Centrifuge app
+
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![CircleCI](https://circleci.com/gh/Zondax/ledger-centrifuge.svg?style=shield)](https://circleci.com/gh/Zondax/ledger-centrifuge)
```

Comment:
Can you change the badge to point to github actions?

Reply:
Done! I think that master branch should be renamed to main as well!